cmd/kubeadm/app/constants/constants.go

```diff
@@ -425,7 +429,7 @@ var (
 		14: "3.3.10",
 		15: "3.3.10",
 		16: "3.3.10",
-		17: "3.3.10",
+		17: "3.3.15-0",
```

Comment:
we could have done that for both 16 and 17 in @dims PR.

Reply:
Right, I noticed the same thing while writing this PR up. I'll simplify this PR and we can use it to amend.

Reply:
Ok, I'll hold it on until #82199 lands in.
/hold